GPflowOpt/acquisition.py

```diff
@@ -238,8 +244,8 @@ class ExpectedImprovement(Acquisition):
        \\alpha(\\mathbf x_{\\star}) = \\int \\max(f_{\\min} - f_{\\star}, 0) \\, p(\\mathbf f^{\\star}\\,|\\, \\mathbf x, \\mathbf y, \\mathbf x_{\\star} ) \\, d\\mathbf f_{\\star}
     """
 
-    def __init__(self, model):
-        super(ExpectedImprovement, self).__init__(model)
+    def __init__(self, model, optimize_restarts=5):
```

Comment:
"""
 
         :param model: GPflow model (single output) for computing the Expected Improvement
"""